src/ui/settings/ViewPrivacySecurity.qml

```diff
@@ -55,19 +55,14 @@ Item {
                 anchors.rightMargin: VPNTheme.theme.windowMargin
             }
 
-            VPNVerticalSpacer {
-                height: 1
-                width: parent.width
-            }
-
             VPNCheckBoxRow {
                 id: dataCollection
                 objectName: "dataCollection"
                 width: parent.width - VPNTheme.theme.windowMargin
                 anchors.left: parent.left
                 anchors.right: parent.right
                 anchors.rightMargin: VPNTheme.theme.windowMargin
-
+                showDivider: false
```

Comment:
Similar question as below but if `crashOptInCheckBox` is only visible on certain platforms, then this should be visible if `!crashOptInCheckBox.visible`.

Reply:
Data collection is for glean, not crash reporting.  It is available everywhere.

Reply:
I've updated this now that I understand the comments.